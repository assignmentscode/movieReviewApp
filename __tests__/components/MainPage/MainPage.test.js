import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Mainpage from '../../../components/MainPage/MainPage';

describe('MainPage', () => {
  it('snapshot should match the previous snapshot', () => {
    const tree = renderer.create(<Mainpage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
