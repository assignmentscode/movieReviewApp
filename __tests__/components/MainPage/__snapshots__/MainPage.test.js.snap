// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPage snapshot should match the previous snapshot 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "green",
        "padding": 8,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Object {
          "alignSelf": "center",
          "fontWeight": "bold",
        }
      }
    >
      Movie Reviews
    </Text>
  </View>
  <RCTScrollView>
    <View />
  </RCTScrollView>
</RCTSafeAreaView>
`;
